android/feature-login/src/main/java/io/newm/feature/login/screen/createaccount/CreateAccountPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package io.newm.feature.login.screen.createaccount
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import com.slack.circuit.retained.rememberRetained
+import com.slack.circuit.runtime.Navigator
+import com.slack.circuit.runtime.presenter.Presenter
+import io.newm.feature.login.screen.TextFieldState
+import io.newm.feature.login.screen.createaccount.signupform.SignupFormUiEvent
+import io.newm.feature.login.screen.password.ConfirmPasswordState
+import io.newm.feature.login.screen.password.PasswordState
+
+class CreateAccountPresenter : Presenter<CreateAccountUiState> {
```

Comment:
Never heard about this Circuit library before. They should have use a different name instead of `Presenter`, because of all the old MVP architecture baggage.